server/service/integration_mdm_test.go

```diff
@@ -5316,7 +5316,178 @@ func (s *integrationMDMTestSuite) TestGetPoliciesRequestWithNotElegibleHost() {
+				<ac:AdditionalContext xmlns="http://schemas.xmlsoap.org/ws/2006/12/authorization">
+					<ac:ContextItem Name="UXInitiated">
+					<ac:Value>false</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="HWDevID">
+					<ac:Value>CF1D12AA5AE42E47D52465E9A71316CAF3AFCC1D3088F230F4D50B371FB2256F</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="Locale">
+					<ac:Value>en-US</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="TargetedUserLoggedIn">
+					<ac:Value>true</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="OSEdition">
+					<ac:Value>48</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="DeviceName">
+					<ac:Value>DESKTOP-0C89RC0</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="MAC">
+					<ac:Value>01-1C-29-7B-3E-1C</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="MAC">
+					<ac:Value>01-0C-21-7B-3E-52</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="DeviceID">
+					<ac:Value>AB157C3A18778F4FB21E2739066C1F27</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="EnrollmentType">
+					<ac:Value>Full</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="DeviceType">
+					<ac:Value>CIMClient_Windows</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="OSVersion">
+					<ac:Value>10.0.19045.2965</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="ApplicationVersion">
+					<ac:Value>10.0.19045.1965</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="NotInOobe">
+					<ac:Value>false</ac:Value>
+					</ac:ContextItem>
+					<ac:ContextItem Name="RequestVersion">
+					<ac:Value>5.0</ac:Value>
+					</ac:ContextItem>
+				</ac:AdditionalContext>
+				</wst:RequestSecurityToken>
+			</s:Body>
+			</s:Envelope>
+		`)
+
+	resp := s.DoRaw("POST", microsoft_mdm.MDE2EnrollPath, requestBytes, http.StatusOK)
```

Comment:
Not necessarily for this PR, but since we start having a good number of SOAP-related tests, might be worth adding an `s.DoSOAP` like we have an `s.DoJSON`, and get the unmarshaled SOAP for additional assertions on the response.